artifacts/Dockerfile

```diff
@@ -118,6 +121,10 @@ COPY /src/deployment/docker/id-authentication/* ${ida_sh_path}/
 
 COPY /src/hsm/client.zip ${hsm_client_path}/
 
+COPY /src/mosip-plugins/sign-in-with-esignet/sign-in-with-esignet.zip ${mosip_plugins_zip_path}/
+
+COPY /src/mosip-plugins/secure-biometric-interface-integrator/secure-biometric-interface-integrator.zip ${mosip_plugins_zip_path}/
```

Comment:
As discussed there is no internal service is depending on this, lets not keep it part of the artifactory

Reply:
Removed the sbi integrator plugin from the artifactory